File: Examples/Python/MongoDB/REST/rest_response.py
class REST_Response:
    def __init__(self, status_code: int, content_type: str, body: str):
        self.status_code = status_code
        self.content_type = content_type
        self.body = body

    def __repr__(self):
        return f"RestResponse(status_code={self.status_code}, content_type='{self.content_type}', body='{self.body}')"

    def to_dict(self):
        return {
            "status_code": self.status_code,
            "content_type": self.content_type,
            "body": self.body
        }

File: Examples/PHP/Containers/Laravel-MongoDB-REST/app/Services/MongoDB/REST/REST_Response.php
<?php

namespace App\Services\MongoDB\REST;

class REST_Response
{
    private $statusCode;
    private $contentType;
    private $body;

    public function __construct($statusCode, $contentType, $body)
    {
        $this->statusCode = $statusCode;
        $this->contentType = $contentType;
        $this->body = $body;
    }

    public function getStatusCode()
    {
        return $this->statusCode;
    }

    public function getContentType()
    {
        return $this->contentType;
    }

    public function getBody()
    {
        return $this->body;
    }

    public function __toString()
    {
        return sprintf(
            "RestResponse(status_code=%d, content_type='%s', body='%s')",
            $this->statusCode,
            $this->contentType,
            $this->body
        );
    }

    public function toArray()
    {
        return [
            'status_code' => $this->statusCode,
            'content_type' => $this->contentType,
            'body' => $this->body
        ];
    }
}